defaults/jq/semconv.jq

```diff
@@ -0,0 +1,102 @@
+def semconv_group_attributes_by_namespace:
+    group_by(.namespace)
+    | map({ namespace: .[0].namespace, attributes: . | sort_by(.name) });
+
+#####################
+# Attribute functions
+#####################
+def semconv_attributes($options):
```

Comment:
Nit: Can you document the purpose of these functions in JQ, including options supported?

Reply:
I will

Reply:
Done